client/package.json

```diff
@@ -16,7 +16,7 @@
     "@hedgedoc/html-to-react": "^1.4.5",
     "@lexical/react": "^0.11.0",
     "@react-google-maps/api": "^2.17.0",
-    "@refugies-info/api-types": "1.0.51",
+    "@refugies-info/api-types": "*",
```

Comment:
That shouldn't be the case because using `*` just means the build is going to use the version of `@refugies-info/api-types` that exists in the current branch.  So as long as the branch builds `api-types`, `server` and `client` should be compatible. Am I missing something?